Sources/CSVDealer/CSVDealer.swift

```diff
@@ -54,12 +54,18 @@ public struct CSVParser<T: CSVConvertible> {
         return (headers, nonHeaderRows)
     }
     
-    public func parse() throws -> [T] {
+    public func parse(lineCountToParse: Int? = nil) throws -> [T] {
```

Comment:
we should also change the param name so callers know it will return a random set of rows.